overrides/groovy/postInit/Post-Initial/Main/General/Early-Game/earlygame.groovy

```diff
@@ -9,6 +9,58 @@ import net.minecraftforge.fluids.FluidStack
 import static com.nomiceu.nomilabs.groovy.GroovyHelpers.RecyclingHelpers.*
 import static gregtech.api.GTValues.*
 
+// Allow treated wood sticks in (most) stick recipes
+ore('stickWood').add(metaitem('stickTreatedWood'))
```

Comment:
By right this shouldn't be allowed, treated wood is a separate material from normal wood (similar to iron/wrought iron).
If the players think this is a reasonable QoL it can go in though

Reply:
I think this will create more annoyances than it solves. Creosote is much less free than wood and you might accidentally waste treated sticks with jei's + button